lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -957,6 +957,14 @@ def cmd_load_help
     print_line
   end
 
+  def log_on_timeout_error(message)
```

Comment:
I believe this will be part of our public API that users shouldn't really be calling - it seems more like an private/internal implementation. Let's move this down to below to the `protected` section

https://github.com/rapid7/metasploit-framework/blob/cd0ccd1716cd5ae71be33c618dd613268932f0cf/lib/msf/ui/console/command_dispatcher/core.rb#L2582C1-L2583C1